steam_app/controllers/GamesViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ViewController.swift
+//  steam_app
+//
+//  Created by Khumoyiddin on 28/12/22.
+//
+
+import UIKit
+
+class GamesViewController: UIViewController {
+    
+    @IBOutlet weak var tableView: UITableView!
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let nib = UINib(nibName: "GamesTableViewCell", bundle: nil)
+        
+        tableView.register(nib, forCellReuseIdentifier: "GamesTableViewCell")
+        tableView.delegate = self
+        tableView.dataSource = self
+    }
+
+}
+
+extension GamesViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        print("tapping on cell")
+    }
+}
+
+
+extension GamesViewController: UITableViewDataSource {
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return LocalCache.gamesList.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "GamesTableViewCell", for: indexPath) as! GamesTableViewCell
+        let game = LocalCache.gamesList[indexPath.row]
+        cell.titleLabel.text = game.title
```

Comment:
it is better not to access to UI elements directly. What you could to is to create a method in the cell and call it in the controller to hide other unnecessary UI elements usage